### PR TITLE
modified an example of face_detection_pytorch

### PR DESCRIPTION
**Pull request by kmotohas**
as discussed in https://github.com/KonduitAI/konduit-serving/issues/17

- output variable name was "image" before but it is "num_boxes" now
- fixed issue on importing face detector modules

### CODE REVIEW

File: python/examples/face_detection_pytorch/detect_image.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 from vision.ssd.mb_tiny_RFB_fd import create_Mb_Tiny_RFB_fd, create_Mb_Tiny_RFB_fd_predictor
 from utils import base64_to_ndarray
 
+
 threshold = 0.7
 candidate_size = 1500
 define_img_size(640)
 test_device = 'cpu'
 
+work_dir = os.path.abspath('./Ultra-Light-Fast-Generic-Face-Detector-1MB')
```
Comment (kmotohas) [inline python/examples/face_detection_pytorch/detect_image.py]: The following error happens in my environment. Do you see what is wrong?
```
No module named 'vision'
Traceback (most recent call last):
  File "<string>", line 12, in <module>
ModuleNotFoundError: No module named 'vision'
```

**Comment by maxpumperla** [comment]
@kmotohas I fixed a small indentation error for you.

the problem is I had to merge master into your branch, and now pretty much all Python tests fail. :D

```11:28:14.668 [vert.x-worker-thread-0] ERROR a.k.s.c.PipelineRouteDefiner - Failed to initialize. Shutting down.
java.lang.NullPointerException: null
	at ai.konduit.serving.executioner.Pipeline.getPipeline(Pipeline.java:65)
	at ai.konduit.serving.executioner.PipelineExecutioner.init(PipelineExecutioner.java:127)
	at ai.konduit.serving.configprovider.PipelineRouteDefiner.defineRoutes(PipelineRouteDefiner.java:451)
	at ai.konduit.serving.verticles.inference.InferenceVerticle.init(InferenceVerticle.java:83)
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$8(DeploymentManager.java:553)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369)
	at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:35)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
11:28:22.128 [vert.x-worker-thread-7] ERROR a.k.s.c.PipelineRouteDefiner - Request failed with cause 
java.lang.NullPointerException: null
	at ai.konduit.serving.configprovider.PipelineRouteDefiner.getAdapterMap(PipelineRouteDefiner.java:468)
	at ai.konduit.serving.configprovider.PipelineRouteDefiner.lambda$defineRoutes$5(PipelineRouteDefiner.java:267)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:986)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:131)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:130)
	at io.vertx.ext.web.handler.impl.BodyHandlerImpl$BHandler.doEnd(BodyHandlerImpl.java:296)
	at io.vertx.ext.web.handler.impl.BodyHandlerImpl$BHandler.uploadEnded(BodyHandlerImpl.java:265)
	at io.vertx.ext.web.handler.impl.BodyHandlerImpl$BHandler.lambda$null$1(BodyHandlerImpl.java:204)
	at io.vertx.core.http.impl.HttpServerFileUploadImpl.handleEnd(HttpServerFileUploadImpl.java:87)
	at io.vertx.core.http.impl.HttpServerFileUploadImpl.lambda$null$1(HttpServerFileUploadImpl.java:178)
	at io.vertx.core.impl.ContextImpl.lambda$null$0(ContextImpl.java:330)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369)
	at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:35)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by maxpumperla** [comment]
to be clear, I think your fix will work, but we need to address this issue first.

File: python/examples/face_detection_pytorch/detect_image.py (hunk 1 above)
Comment (maxpumperla) [reply python/examples/face_detection_pytorch/detect_image.py]: oh, you're right. that's why you add the path first. I was under the impression that was a dependency installed through `pip`.

**Comment by maxpumperla** [comment]
@kmotohas it works now. my konduit.jar was painfully outdated, there is in fact no error.

**Pull request merged**